ansible/roles/mn-evo-services/templates/dapi-nginx.conf.j2

```diff
@@ -101,6 +107,8 @@ server {
         }
 
         proxy_pass http://{{ private_ip }}:3020$request_uri;
+        proxy_http_version 1.1;
+        proxy_hide_header access-control-allow-origin;
```

Comment:
Sorry, I don't get how it works. What is the point to add `add_header 'Access-Control-Allow-Origin' '*';` for `POST` and then hide it?

Reply:
It's hiding Access-Control-Allow-Origin received from envoy. If we don't hide - we'll get double header, even if the value is the same. And another thing that I don't know how to fix - envoy sends this header equals 'null' instead of '*'.

Reply:
In this case, probably we just need to remove cors configuration from envoy?

Reply:
Yep, if we don't want to use envoy directly.